test/harmonytest.js

```diff
@@ -18524,9 +18524,9 @@ var harmonyTestFixture = {
         },
 
         'func(...a, b)': {
-            index: 9,
+            index: 10,
             lineNumber: 1,
-            column: 10,
+            column: 11,
```

Comment:
> What does this mean for `func(...a,)`? That one isn't an error right?

I think it should be a parse error -- nothing really can follow the rest args. @jeffmo, did you mention this in the pre-spec?